tools/aml-language-server

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+ARCS_DIR="$(dirname "$(dirname "$0")")"
```

Comment:
Oh, actually I forgot to add @mykmartin to the review so I could ask exactly that question.
@mykmartin is there a better method for this? I'm not sure it will work on windows either.

By way of explanation: it's supposed to fix the problem of running the tools/aml-language-server from somewhere else, while not knowing where the arcs repo is stored.

Reply:
I also don't know about windows compatibility, but you could just do this: `cd "$(dirname "$0")"/..`

Note that will fail if someone uses a symlink to the aml-language-server script, though. On un*x systems `readlink` is useful but again, not sure about windows.